test/e2e/source_list_crd_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 The Knative Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or im
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build e2e
+// +build !serving
+// +build !project_admin
```

Comment:
I would include some comment to the test, that this test only works for cluster admin and essentially the comment that you added to the issue. That way people are more aware that this test is special. 

We should also open an issue to fix this in the kn code and add a link to the issue here.